conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ConjureHandler.java

```diff
@@ -45,7 +51,7 @@
             // Bearer token and trace handler must execute prior to the exception
             // to provide user and trace information on exceptions.
             (endpoint, handler) -> new BearerTokenLoggingHandler(handler),
-            TraceHandler::new,
+            (endpoint, handler) -> new TraceHandler(endpoint.method + " " + endpoint.template, handler),
```

Comment:
EndpointDetails is a private internal structure, the TraceHandler just needs an operation name, I'd prefer not to leak the EndpintDetails abstraction here

Reply:
I'd like to avoid that, by design I don't want the trace handler to be smart, it should be impossible to make it calculate operation names per request because that can leak unsafe data.

Reply:
👍